src/org/montclairrobotics/sprocket/control/ArcadeDriveInput.java

```diff
@@ -1,53 +1,60 @@
 package org.montclairrobotics.sprocket.control;
 
-import edu.wpi.first.wpilibj.Joystick;
+import org.montclairrobotics.sprocket.drive.DriveTrainTarget;
 import org.montclairrobotics.sprocket.drive.MotorInputType;
-import org.montclairrobotics.sprocket.drive.DriveTrainInput;
+import org.montclairrobotics.sprocket.geometry.Angle;
 import org.montclairrobotics.sprocket.geometry.Degrees;
 import org.montclairrobotics.sprocket.geometry.Inch;
 import org.montclairrobotics.sprocket.geometry.Polar;
 import org.montclairrobotics.sprocket.geometry.Vector;
+import org.montclairrobotics.sprocket.geometry.XY;
+import org.montclairrobotics.sprocket.pipeline.Step;
+
+import edu.wpi.first.wpilibj.Joystick;
 
-public class ArcadeDriveInput extends DriveTrainInput {
+public class ArcadeDriveInput implements Step<DriveTrainTarget> {
```

Comment:
Maybe a better question to ask is if human/autonomus mode inputs should be distinct from other steps in the pipeline.